tests/ActiveRecordTest.php

```diff
@@ -36,7 +44,7 @@ public function testFind()
 
         // find all
         $articles = ArticleIndex::find()->all();
-        $this->assertEquals(2, count($articles));
+        $this->assertEquals($this->pagination->defaultPageSize, count($articles));
```

Comment:
This is not a proper assertion.
Exact number match should be used as it was before.